_data/team.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: "Oskar Thoren"
+  shortName: "oskarth"
+  image: "oskar.png"
+  statusLink: https://status.im/
```

Comment:
You sure you wanna just link people to status when they click on your name? Doesn’t really make sense imo. Might as well remove it then. Along with status name

Reply:
This is a placeholder to show how it renders, I'll either remove or update statusName and link.

Reply:
I'd vote for remove it, but up to you.

Reply:
https://nimbus.team/

Reply:
we link to our status keys so that we're contactable over status - seems fitting (ie https://join.status.im/user/0x049bc24b377c7e97bb94ab993e48fcbfe38f96aad43a42418d7f33f5a99f6f89e2af91875bfe045205771f28c93be50380cdfe5772201fde7019d57f7ead9af9b5)